src/features/shared-link-settings-modal/AllowDownloadSection.js

```diff
@@ -35,23 +38,38 @@ const AllowDownloadSection = ({
         </div>
     );
 
+    const label = classification ? `"${classification}"` : null;
+    const tooltipMessage = { ...messages.directDownloadBlockedByAccessPolicy, values: { label } };
```

Comment:
If label can be null, then that means there will be a situation where the message will be `Download for this content is disabled due to the classification`. Do we want to show the tooltip if label is null?